### ISSUE TITLE
Replicate your result

### ISSUE THREAD

**Issue by glmanhtu** [wtomin/Multitask-Emotion-Recognition-with-Incomplete-Labels#11]
Hello Didan,

I'm trying to reproduce the training student CNN models based on the provided teacher model (which seems to be easier than training everything from scratch). However, I'm unable to get the same level with your results on the AffWild 2 validation set. Could you have a look to see if there is any mis-configured parameters ?. 

The command to train is: `python3.7 train.py --force_balance --name image_size_112_n_students_5_pretrained --image_size 112 --batch_size 64 --print_freq_s 240 --pretrained_teacher_model /home/mvu/Documents/pretrained/FG2020-ABAW/Multitask-CNN/net_epoch_teacher_id_resnet50.pth` 

My evaluation result:

> Merged First method EXPR Validation 08:21: Eval_0 0.3389 Eval_1 0.4597 eval_res 0.3787
Merged First method AU Validation 08:21: Eval_0 0.1828 Eval_1 0.9453 eval_res 0.5640
Merged First method VA Validation 08:21: Eval_0 0.3517 Eval_1 0.4941 eval_res 0.8457
Merged Second method EXPR Validation 08:21: Eval_0 0.3386 Eval_1 0.4552 eval_res 0.3771
The best AU thresholds over models: [0.1823972  0.07081039 0.13026756 0.04179615 0.3001067  0.07159575 0.04400642 0.19260739]
Merged Second method AU Validation 08:21: Eval_0 0.3241 Eval_1 0.9305 eval_res 0.6273
Merged Second method VA Validation 08:21: Eval_0 0.3100 Eval_1 0.4623 eval_res 0.7724

The AffWild-2 dataset is the cropped, aligned version from the organizer. The training and evaluating log are in the attachment
[multi-cnn-pretrained.log](https://github.com/wtomin/Multitask-Emotion-Recognition-with-Incomplete-Labels/files/5909823/multi-cnn-pretrained.log)
